### PR TITLE
Contact Height  and Width  changed

### PR DESCRIPTION
**Pull request by XeroDays**

### CODE REVIEW

**Review by Sikander-shah007 — COMMENTED** [review]
yara code maza nishta

**Review by Sikander-shah007 — CHANGES_REQUESTED** [review]
max with property

File: style.css [hunk 1]
```diff
@@ -16,4 +16,6 @@ footer {
 .content {
     min-height: 200px;
     background: blue;
+    width: 10px;
+    margin: 10px;
```
Comment (XeroDays) [inline style.css:20]: mara set d yar

**Comment by XeroDays** [comment]
mara set  d  yar..

**Review by Sikander-shah007 — APPROVED** [review]
Good Work tabahi coder

**Pull request merged**